### PR TITLE
create a new navigation menu

### PR DESCRIPTION
**Pull request by Nahid01920**

### CODE REVIEW

**Review by InfinityLabsCode — CHANGES_REQUESTED** [review]
Add prettier in your vs code and format the code

File: styles/globals.scss [hunk 1]
```diff
@@ -15,3 +15,13 @@
 body {
   font-family: 'PPNeueMachina';
 }
+.login-area {
```
Comment (InfinityLabsCode) [inline styles/globals.scss]: Why using css files instead of tailwind?

File: pages/createpoll.tsx [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
+  });
 
   return (
     <>
       <Head>
         <title>SecurePoll</title>
       </Head>
       <Header />
+      <Pagebody />
+      <Fotter />
```
Comment (InfinityLabsCode) [inline pages/createpoll.tsx:19]: Footer spelling is wrong.

Comment (Nahid01920) [reply pages/createpoll.tsx:19]: fixed

**Review by InfinityLabsCode — APPROVED** [review]

**Pull request merged**